### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by mattuylee**
Only tested under linux.

### CODE REVIEW

**Comment by mattuylee** [comment]
I am adding translation of Chinese, but if I use launguage tag "zh_CN", the translation check will fail because translation file name not ends with "cn".  
I changed to use "zh" to pass the check, but maybe it's a problem when someone else adding translation for other Chinese such as "zh_TW".

Because of different word order, sometimes blank translation is needed, but this is treated as unfinished translation.

And, in Chinese, we use "。" instead of ".", so there are lots of annoying warnings...

**Comment by Abestanis** [comment]
Hey, thank you very much for adding the translation and opting into actively maintaining it.

> the translation check will fail because translation file name not ends with "cn".

Sorry about that, that was an oversight on my part. If you rebase on the latest master it should be fixed. I agree that the file names should use the long language tag to prevent problems in the future, could you please change them back?

> there are lots of annoying warnings

Yes, unfortunately this is unavoidable, because these checks assume that the language is similar to English. Fortunately, there is a way to suppress the warnings, but it requires a bit more effort to set up:

Go through each of the warnings and check if they are valid. If you think the warning is invalid, you can suppress it by adding a `translatorcomment` with `checkTranslation ignore: ...` as the content to the translation with the invalid warning like this:
https://github.com/gyunaev/birdtray/blob/dad3c9b550468fee94868746bef46474d4b0fc30/src/translations/main_de.ts#L366-L370

Each warning type has an id which is included in brackets at the end of the message:
Warning with warning id in brackets at the end

This id needs to be in the `translatorcomment` in the place of the `...`. If you need to suppress multiple warnings for one translation, you can add multiple warning ids by separating them with a comma.

**Review by Abestanis — COMMENTED** [review]
Awesome, thank you very much. I hope dealing with the translation checker wasn't too much work.

Looks very good, I left two comments for you to look at, once you looked at them I would be ready to merge this.

File: src/translations/main_zh_cn.ts
Comment (mattuylee) [reply src/translations/main_zh_cn.ts:777]: yeah, I think maybe the `...` can be a little confusing. For the `Snooze for...` menu, there is a sub-menu so I included that.

File: src/translations/main_zh_cn.ts
Comment (mattuylee) [reply src/translations/main_zh_cn.ts:37]: Okay. I'll change them all.

File: src/translations/main_zh_cn.ts
Comment (Abestanis) [reply src/translations/main_zh_cn.ts:777]: In my opinion the settings dialog is like a sub-menu for the `Settings...` button, but I will leave it up to you. If you don't want to change it, just mark this conversation as resolved.

Comment (mattuylee) [reply src/translations/main_zh_cn.ts]: Okay, you are right.

**Comment by mattuylee** [comment]
> Awesome, thank you very much. I hope dealing with the translation checker wasn't too much work.
> 
> Looks very good, I left two comments for you to look at, once you looked at them I would be ready to merge this.

Adding translationcomment data didn't take too much time. But maybe a tip could be more apparent? So that translators know how to fix when checks failed.

**Review by Abestanis — APPROVED** [review]
Thank you!

> But maybe a tip could be more apparent?

Yeah, I will add that to the wiki page.

**Pull request merged**
